minerl/Malmo/Schemas/Mission.xsd

```diff
@@ -235,6 +235,14 @@
                         <xs:attribute name="foodSaturation" type="xs:float" use="optional" />
                     </xs:complexType>
                 </xs:element>
+                <xs:element name="LowLevelInputs" type="xs:boolean" default="false" minOccurs="0" maxOccurs="1">
+                    <xs:annotation>
+                        <xs:documentation>
+                            If set to true, inputs to malmo are injected instead of keyboard and mouse (instead of higher level commands)
```

Comment:
Maybe "If set to true, inputs to malmo are disabled and instead given via a virtual keyboard and mouse (instead of higher level commands)"